torchvision/prototype/transforms/_color.py

```diff
@@ -105,9 +105,9 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
         return dict(
             zip(
                 ["brightness", "contrast1", "saturation", "hue", "contrast2"],
-                torch.rand(6) < self.p,
+                (torch.rand(6) < self.p).tolist(),
             ),
-            contrast_before=torch.rand(()) < 0.5,
+            contrast_before=bool(torch.rand(()) < 0.5),
```

Comment:
This and the one above are mostly cosmetic to have an actual bool rather than a boolean tensor scalar. This has no effect on the runtime since `if x` invokes `bool(x)` anyway if `x` is not a boolean.